ops/gulp/icons-generate.js

```diff
@@ -0,0 +1,24 @@
+const gulp = require('gulp');
+const svgmin = require('gulp-svgmin');
+const svgSprite = require("gulp-svg-sprites");
+
+// svgSprite configuration
+const config = {
+    templates: { css: require("fs").readFileSync("./src/icons/techne-icons-template.css", "utf-8") },
+    selector: "hyicon-%f",
+    common: "hyicon",
+    svg: { sprite: "css/techne-icons.svg" },
+    cssFile: "css/techne-icons.css",
+    preview: {sprite: "techne-icons.html"}
```

Comment:
Keep the names for the generated file simple, no `techne-` prefix. Then do the prefixing in the `pkg` tasks along with the other files. So if we decide to change to `tn-all.css` it'll be easier to change that as part of the packaging tasks instead of having to dip into this file.